website/src/_includes/flink_java_pipeline_options.html

```diff
@@ -1,150 +0,0 @@
-<!--
```

Comment:
Heads-up, this file is automatically generated as part of the Flink build target `:runners:flink:1.10:generatePipelineOptionsTableJava`.

Reply:
Please ping me, I can migrate the task and the generator to a different format.